newjersey/navigator.business.nj.gov
# feat: [#188006371] add housing hotel motel license status check

## Description

Adds a hotel/motel license status check, that functions similarly to the elevator registration status check



### Ticket



This pull request resolves [#188006371](https://www.pivotaltracker.com/story/show/188006371).

### Approach



### Steps to Test
Create an account in the Lodging industry
Answer "Yes" to the hotel/motel non-essential question
See the task in Step 4
Use the Q/A testing doc to get addresses to test



### Notes



## Code author checklist
